bcgov/House-Innovations-HOUSNAV
# HOUSNAV-186: added and updated landing pages and IA

[HOUSNAV-186](https://hous-bssb.atlassian.net/browse/HOUSNAV-186)

## Overview & Purpose
Refactor the home page to match the design and creating new building type landing page placeholders

## Summary of Changes
- Created a building type paradigm and updated the website architecture to match

## Testing
- Go to landing page and see new design
- Click on Single Dwelling button and see the old landing page with new intro content
- Go back to home and click the Multi-Dwelling button and see the placeholder page

## Checklist
